templates/password-auth-ac.erb

```diff
@@ -0,0 +1,21 @@
+# This file is being maintained by Puppet.
+# DO NOT EDIT
+# Auth
```

Comment:
@nbentoumi please add this extra line and you have integrated all corrections thath @ghoneycutt mentioned